mmv1/products/compute/Image.yaml

```diff
@@ -368,4 +368,4 @@ properties:
       - :READY
       - :PENDING
       - :FAILED
-  # State is not applicable for state convergence.
+# State is not applicable for state convergence.
```

Comment:
Looking at the [old api.yaml](https://github.com/GoogleCloudPlatform/magic-modules/blob/d68ff456c5976466db9865a3d7329dcb9d4650df/mmv1/products/compute/api.yaml) this came from, it seems like it's possibly supposed to capture that this field is being intentionally excluded from the resource; not sure what indentation makes the most sense here.